### PR TITLE
added important dates to lannding page

### PR DESCRIPTION
**Pull request by YuYing-Liang**
## Overview

- Resolves #IEEE-156
- Added dates to landing page
- there is still some stuff that we need to clarify from logistics though

## Unit Tests Created

- <Briefly describe unit tests that were created>

## Steps to QA

- <Steps for a reviewer to QA your work>

### CODE REVIEW

**Review by Mustaballer — COMMENTED** [review]
Looks good!

File: hackathon_site/hackathon_site/settings/__init__.py [hunk 1]
```diff
@@ -263,10 +263,10 @@
 DEFAULT_FROM_EMAIL = "webmaster@localhost"
```
Comment (Leo6Leo) [inline hackathon_site/hackathon_site/settings/__init__.py]: Do we need to change this?

**Review by Leo6Leo — COMMENTED** [review]
Other than the one I commented, Looks good to me as well!!

File: hackathon_site/hackathon_site/settings/__init__.py (hunk 1 above)
Comment (YuYing-Liang) [reply hackathon_site/hackathon_site/settings/__init__.py]: changed in another PR for IEEE-154 (just got merged)

File: hackathon_site/hackathon_site/settings/__init__.py [hunk 2]
```diff
@@ -272,10 +272,10 @@
...
 CONTACT_EMAIL = DEFAULT_FROM_EMAIL
 
-REGISTRATION_OPEN_DATE = datetime(2020, 9, 1, tzinfo=TZ_INFO)
-REGISTRATION_CLOSE_DATE = datetime(2023, 9, 30, tzinfo=TZ_INFO)
-EVENT_START_DATE = datetime(2023, 10, 10, 10, 0, 0, tzinfo=TZ_INFO)
-EVENT_END_DATE = datetime(2023, 10, 11, 17, 0, 0, tzinfo=TZ_INFO)
+REGISTRATION_OPEN_DATE = datetime(2022, 1, 3, tzinfo=TZ_INFO)
+REGISTRATION_CLOSE_DATE = datetime(2022, 2, 5, tzinfo=TZ_INFO)
+EVENT_START_DATE = datetime(2022, 2, 19, 12, 0, 0, tzinfo=TZ_INFO)
+EVENT_END_DATE = datetime(2022, 2, 20, 12, 0, 0, tzinfo=TZ_INFO)
```
Comment (lisa-sa-li) [inline hackathon_site/hackathon_site/settings/__init__.py:278]: do the `0, 0` indicate hour and time of the day? If so, wouldn't it be something like 9am start and 5pm end

**Review by grahamhoyes — APPROVED** [review]

File: hackathon_site/event/jinja2/event/landing.html [hunk 3]
```diff
@@ -108,38 +108,39 @@ <h3 class="center" id="countdownTitle"></h3>
...
                 </div>
 
                 <div class="col s12 m6 l6">
-                    <h5 class="center">Networking Booths</h5>
+                    <h5 class="center">Hardware Pickup 2</h5>
 
                     <div class="scheduleRow">
-                        <p class="scheduleTime">3-6PM</p>
-                        <p>Saturday, February 6th, 2021</p>
+                        <p class="scheduleTime">10:30AM</p>
```
Comment (lisa-sa-li) [inline hackathon_site/event/jinja2/event/landing.html:125]: Is there an end time?

File: hackathon_site/hackathon_site/settings/__init__.py (hunk 2 above)
Comment (lisa-sa-li) [reply hackathon_site/hackathon_site/settings/__init__.py:278]: oh wait I see above its like 12PM -12PM, in that case should it be `12, 0` @grahamhoyes

Comment (YuYing-Liang) [reply hackathon_site/hackathon_site/settings/__init__.py:278]: it's 12, 0 rn the last 0 is seconds i'm pretty sure

File: hackathon_site/event/jinja2/event/landing.html (hunk 3 above)
Comment (YuYing-Liang) [reply hackathon_site/event/jinja2/event/landing.html:125]: logistics said "onwards throughout the day" but the row looks all weird if I type too much in the time column

**Review by lisa-sa-li — APPROVED** [review]

**Pull request merged**